cypress/utils/utils.ts

```diff
@@ -1674,31 +1674,38 @@ export function isRwxEnabled(): boolean {
 // This method is patching
 export function configureRWX(isEnabled = true): void {
     // Patching CR to set value
-    let value = "";
-    if (isEnabled) {
-        value = "true";
-    } else {
-        value = "false";
-    }
-    let command = "";
-    let tackleCr = "tackle=$(oc get tackle --all-namespaces|grep -iv name|awk '{print $2}'); ";
-    let namespace =
-        'namespace=$(oc get tackle --all-namespaces|egrep "tackle|mta"|cut -d " " -f 1); ';
-    command += tackleCr;
-    command += namespace;
-    command += "oc patch tackle ";
-    command += "$tackle ";
-    command += "-n$namespace ";
-    command += "--type merge ";
-    command += `--patch '{"spec":{"rwx_supported": ${value}}}'`;
-    cy.log(command);
-    cy.exec(command).then((result) => {
-        cy.log(result.stderr);
+    cy.url().then(($url) => {
+        const pattern = /-(.*?)\./;
```

Comment:
Please add some comments with example URL to explain here 
Like https://tackle-konveyor-tackle.apps.mig09.rhos-psi.cnv-qe.rhood.us
How you are getting NS from this.